app/src/main/res/values/strings_debug.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="debug_version_gms" translatable="false">Google Play services: %1$s</string>
```

Comment:
Move the content to localization spreadsheet

Reply:
this is just for debugging and it should not confuse or influence the regular translations. that is why they are marked as non-translatable and in a separate file.